content/en/platform/corda/1.6/cenm/release-notes.md

```diff
@@ -1,9 +1,9 @@
 ---
 date: '2023-09-12'
 menu:
-  cenm-1-5:
-    identifier: cenm-1-5-release-notes
-    parent: cenm-1-5-cenm-releases
+  cenm-1-6:
+    identifier: cenm-1-6-release-notes
+    parent: cenm-1-6-cenm-releases
```

Comment:
Might be no harm to remove the contents of this page now to avoid confusion?

Reply:
removed